src/lib/Commands/TestCommand/index.js

```diff
@@ -281,9 +310,28 @@ class TestCommand extends AbstractCommand {
     this._info(c.blue('Using ') + testFile.type + c.blue(' test file ') + testFile.name);
 
     // read/process test code
-    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim();
-    this._codeProcessor.variables.__FILE__ = path.basename(testFile.path);
-    testCode = this._codeProcessor.process(testCode);
+    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim().replace(LINE_AND_FILE_REGEXP, "@{__");
+    let unsupportedValues = {}; // map of syntetic_value : original_value
+    var environmentVars = "";
+    for (var prop in process.env) {
+      if (prop !== BUILD_API_KEY_ENV_VAR) { //deny to access for BUILD_API_KEY_ENV_VAR
+        var propValue = process.env[prop].replace(new RegExp('\\\\', 'g'), "\\\\");
```

Comment:
Two questions here:

1. `new RegExp('\\\\', 'g')` -> `/\\\\/g` ?
1. why not `replace(/\\/g, "\\\\")` ?
1. what about other special chars:
```
str.replace(/\\/g, "\\\\")
    .replace(/\$/g, "\\$")
    .replace(/'/g, "\\'")
    .replace(/"/g, "\\\"");
```